modules/op-release-notes-1-10.adoc

```diff
@@ -176,3 +176,19 @@ With this update, {pipelines-title} General Availability (GA) 1.10.2 is availabl
 === Fixed issues
 
 Before this update, an issue in the Tekton Operator prevented the user from setting the value of the `enable-api-fields` flag to `beta`. This update fixes the issue. Now, you can set the value of the `enable-api-fields` flag to `beta` in the `TektonConfig` CR.
+
+[id="release-notes-1-10-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.3
+
+With this update, {pipelines-title} General Availability (GA) 1.10.3 is available on {product-title} 4.11, 4.12, and 4.13.
+
+[id="new-features-1-10-3_{context}"]
+=== New features
+
+With this update, as a cluster administrator, you can customize the following performance configuration fields in the `TektonConfig` CR based on your needs:
```

Comment:
before this update, operator did not expose the following performance configuration fields, this is now fixed
^^^ something like this